ambry-clustermap/src/main/java/com/github/ambry/clustermap/HelixBootstrapUpgradeUtil.java

```diff
@@ -1072,6 +1074,24 @@ private void addUpdateResources(String dcName, Map<String, Set<String>> partitio
         }
       }
     }
+    // if there are some partitions are disabled, mark whole disabling process complete in Helix PropertyStore
+    if (helixAdminOperation == HelixAdminOperation.DisablePartition && !instancesWithDisabledPartition.isEmpty()) {
+      // create znode under admin configs
+      Properties storeProps = new Properties();
+      storeProps.setProperty("helix.property.store.root.path", "/" + clusterName + "/" + PROPERTYSTORE_STR);
+      HelixPropertyStoreConfig propertyStoreConfig = new HelixPropertyStoreConfig(new VerifiableProperties(storeProps));
+      String zkConnectStr = dataCenterToZkAddress.get(dcName).getZkConnectStrs().get(0);
+      HelixPropertyStore<ZNRecord> helixPropertyStore =
+          CommonUtils.createHelixPropertyStore(zkConnectStr, propertyStoreConfig, null);
```

Comment:
Do we have to call the helixPropertyStore.stop() and close() methods after we are done using the property store? Same comment for its usage in HelixParticipant.

Reply:
Good point. Since this is one-off change, we should close it after updating the property store in Helix.